lib/db/api.js

```diff
@@ -1009,13 +1009,22 @@ export const isRemoveGroupUserAllowed = function(email, targetGroup) {
         }
         return db.connect().then(client => {
             return client.collection('groups').aggregate([
```

Comment:
Are you sure we need aggregate instead of findOne?

Reply:
Yes, because of using setIntersection. I think it s not possible to use set in find

Reply:
It looks like here setIntersection can be replaced with [$in](https://www.mongodb.com/docs/manual/reference/operator/query/in/#mongodb-query-op.-in)

Reply:
simplified